scripts/tools/TaskQueueMaintenance.php

```diff
@@ -0,0 +1,248 @@
+<?php
+
+declare(strict_types=1);
+
+namespace oat\taoTaskQueue\scripts\tools;
+
+use common_persistence_Manager;
+use Laminas\ServiceManager\ServiceLocatorAwareInterface;
+use Laminas\ServiceManager\ServiceLocatorAwareTrait;
+use oat\oatbox\action\Action;
+use oat\oatbox\reporting\Report;
+use oat\tao\model\taskQueue\TaskLogInterface;
+use oat\tao\model\taskQueue\TaskLog\TaskLogFilter;
+use oat\tao\model\taskQueue\TaskLog\Broker\TaskLogBrokerInterface;
+
+
+class TaskQueueMaintenance implements Action, ServiceLocatorAwareInterface
```

Comment:
Why not using ScriptAction? It can help with parseParam and display help

Reply:
Done